src/main/java/org/apache/ibatis/migration/template_environment.properties

```diff
@@ -64,6 +64,9 @@ full_line_delimiter=false
 # but some do.
 auto_commit=false
 
+# If set to true, warnings from the database do not interrupt migrations.
+ignore_warnings=false
```

Comment:
When this PR was merged, the default value was `false`. It was changed in 2f38956